velocity_ramp_time: 0.25
distance_to_hold_angle: 0.5
timeout: 0.5